pytorch_lightning/core/memory.py

```diff
@@ -155,12 +160,22 @@ class ModelSummary(object):
           | Name | Type       | Params | In sizes  | Out sizes
         ------------------------------------------------------------
         0 | net  | Sequential | 132 K  | [10, 256] | [10, 512]
+        --------------------------------------------------------------
+                              Totals
+        Trainable params      132 K
+        Non-trainable params  0
+        Total params          132 K
```

Comment:
I'm thinking maybe swap the two columns so that we don't have a floating "totals" header in the middle. My proposal:
```
132 K     trainable params      
0         non-trainable params
132 K     total params 
```

Reply:
Agreed. I did it that way only because others do the same way. Updating it right now.